src/components/Accordion/Accordion.module.css

```diff
@@ -0,0 +1,75 @@
+.sbui-accordion-container {
+  @apply flex flex-col rounded-md;
+}
+
+.sbui-accordion-container--bordered {
+  @apply bg-white dark:bg-dark-700;
+  border: 1px solid #e0e0e0;
+}
+
+.dark .sbui-accordion-container--bordered {
+  border: 1px solid #2a2a2a;
+}
+
+.sbui-accordion-item {
+  @apply w-full;
+}
+
+.sbui-accordion-item__button {
+  @apply flex justify-between w-full text-left cursor-pointer p-2 border-0 font-medium text-base rounded-t-md bg-transparent;
+
+  @apply border-gray-200 text-gray-500 hover:text-gray-600;
+  @apply dark:border-gray-600 dark:text-dark-200 dark:hover:text-white;
+
+  font-family: inherit;
+  font-weight: inherit;
+}
+
+.sbui-accordion-item__button .sbui-typography {
+  @apply w-full max-w-none;
+}
+
+.sbui-accordion-item__button .sbui-typography:last-child {
+  @apply pl-2;
+}
+
+.sbui-accordion-container--bordered
+  .sbui-accordion-item__button:not(:first-child) {
+  @apply rounded-none;
+  border-top: 1px solid #e0e0e0;
+}
+
+.dark
+  .sbui-accordion-container--bordered
+  .sbui-accordion-item__button:not(:first-child) {
+  @apply rounded-none;
+  border-top: 1px solid #2a2a2a;
+}
+
+.sbui-accordion-item__panel {
+  @apply p-2;
```

Comment:
small request - could we use `px-6` padding, we've started using that x axis padding in a lot of the supabase dashboard now.

I imagine in the longterm though, these padding values could be part of a set of theme variables.

Reply:
Done and I see `py-4` being used with it so added that too